tests/functional/network_policy/test_network_policy.py

```
@@ -0,0 +1,453 @@
+
+
+@purple_squad
+class TestNetworkPolicyCompliance(ManageTest):
+    """
+    Category A: RBAC & Conforma compliance tests (tier1).
+    Verify NetworkPolicies exist and CSVs declare proper RBAC.
+    """
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-6900")
+    def test_network_policies_exist(self, network_policies_present):
```

Ruff (0.16.0)

[warning] Unused method argument: `network_policies_present`

(ARG002)